server/app/boards.go

```diff
@@ -20,6 +20,9 @@ var (
 	ErrInsufficientLicense    = errors.New("appropriate license required")
 )
 
+const linkBoardMessage = "@%s linked Board [%s](%s) with this channel"
+const unlinkBoardMessage = "@%s unlinked Board [%s](%s) with this channel"
+
 func (a *App) GetBoard(boardID string) (*model.Board, error) {
 	board, err := a.store.GetBoard(boardID)
 	if model.IsErrNotFound(err) {
```

Comment:
Nothing to do with this PR, but returning nil,nil here is going a crash at some point.  I'll fix.